middleware/middleware_test.go

```diff
@@ -57,9 +58,9 @@ func TestMaxConcurrentRequestsMiddleware(t *testing.T) {
 
 					handlerToTest.ServeHTTP(respRecorder, req)
 					if respRecorder.Code == 200 {
-						resp200++
+						atomic.AddInt32(&resp200, 1)
 					} else if respRecorder.Code == 503 {
-						resp503++
+						atomic.AddInt32(&resp503, 1)
 					} else {
 						randomResp++
```

Comment:
Perhpas you have forgotten the atomic part here :slightly_smiling_face:

Reply:
Oops, fixed.